docs-developer/CHANGELOG-formats.md

```diff
@@ -6,6 +6,10 @@ Note that this is not an exhaustive list. Processed profile format upgraders can
 
 ## Processed profile format
 
+### Version 50
+
+The serialized format now stores sample and counter sample times as time deltas instead of absolute timestamps to reduce the JSON size. The unserialized version is unchanged.
```

Comment:
If I am somebody who outputs profiles, do I have to change my output?

We need to make a decision about what we want to say in this documentation: Should version 50 require the new format, or should version 50 say "both formats are supported"?